onnxruntime/test/featurizers_ops/mean_imputer_transformer_test.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "gtest/gtest.h"
+#include "test/providers/provider_test_utils.h"
+
+#include "Featurizers/MeanImputerFeaturizer.h"
+#include "Featurizers/TestHelpers.h"
+#include "Featurizers/../Archive.h"
+
+namespace NS = Microsoft::Featurizer;
+namespace dft = NS::Featurizers;
+
+namespace onnxruntime {
+namespace test {
+
+namespace {
+template <typename T>
+std::vector<uint8_t> GetStream(const std::vector<typename NS::Traits<T>::nullable_type>& trainingBatches, size_t colIndex) {
```

Comment:
naming convention of colIndex is off.

Reply:
Corrected everywhere

---
In reply to: [373741877](https://github.com/microsoft/onnxruntime/pull/2958#discussion_r373741877) [](ancestors = 373741877)